exporter/awsemfexporter/emf_exporter.go

```diff
@@ -26,6 +26,10 @@ const (
 	// OutputDestination Options
 	outputDestinationCloudWatch = "cloudwatch"
 	outputDestinationStdout     = "stdout"
+
+	// Pulse EMF config
+	pulseMetricNamespace    = "AWS/APM"
```

Comment:
Would be good to call all these vars and methods `awsApm` instead of `pulse`. Rest LGTM!

Reply:
Pulse product team got some feedback from leadership that we should not use APM as naming/branding for Pulse project. We will most likely to remove all `APM` footprint in Pulse soon or later. 

At the moment, I'll keep namespace and log group name to `AWS/APM` in PR and waiting for Product team for new naming suggestions.

Reply:
Oh interesting.. yeah that would need a bunch of changes on the agent side if thats the case. Please let us know once you hear back Min.